compiler/rustc_codegen_llvm/src/llvm/ffi.rs

```diff
@@ -289,6 +295,7 @@ impl RealPredicate {
 /// LLVMTypeKind
 #[derive(Copy, Clone, PartialEq, Debug)]
 #[repr(C)]
+#[allow(dead_code)]
 pub enum TypeKind {
```

Comment:
This suspiciously unused, `TypeKind` from `rustc_codegen_ssa` used around instead.

Reply:
This is #85677